Connecting on 127.0.0.1:4200
node:events:497
      throw er; // Unhandled 'error' event
      ^

Error: spawn C:\Program Files\nodejs\node.exe ENOENT
[90m    at ChildProcess._handle.onexit (node:internal/child_process:286:19)[39m
[90m    at onErrorNT (node:internal/child_process:484:16)[39m
[90m    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)[39m
Emitted 'error' event on ChildProcess instance at:
[90m    at ChildProcess._handle.onexit (node:internal/child_process:292:12)[39m
[90m    at onErrorNT (node:internal/child_process:484:16)[39m
[90m    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)[39m {
  errno: [33m-4058[39m,
  code: [32m'ENOENT'[39m,
  syscall: [32m'spawn C:\\Program Files\\nodejs\\node.exe'[39m,
  path: [32m'C:\\Program Files\\nodejs\\node.exe'[39m,
  spawnargs: [
    [32m'C:\\Users\\hp\\OneDrive\\Bureau\\Food-Delivery-WebApp\\node_modules\\next\\dist\\bin\\next'[39m,
    [32m'dev'[39m,
    [32m'--port=4200'[39m,
    [32m''[39m,
    [32m''[39m
  ]
}

Node.js v20.18.0
